build_tools/autogen_ltc_backend.py

```diff
@@ -236,6 +236,11 @@ def get_opnames(ops):
                 continue
             if base in supported or op in supported:
                 continue
+            # Blacklist new_/_like ops since they are non-differentiable.
+            if re.search('^new_', base) or re.search('^new_', op):
+                continue
+            if re.search('_like$', base) or re.search('_like$', op):
+                continue
```

Comment:
I think this can be simplified a bit
```suggestion
            if any(o.startswith("new_") or o.endswith("_like") for o in (base, op)):
                continue
```